src/helpers/scopeTab.js

```diff
@@ -62,6 +62,10 @@ export default function scopeTab(node, event) {
     x += shiftKey ? -1 : 1;
   }
 
+  // If the tabbable element does not exist,
+  // let the browser control the focus
+  if (typeof tabbable[x] === "undefined") return;
```

Comment:
Maybe, we actually need to focus the first element in this case, otherwise I suspect it can miss radios inputs.

Reply:
Updated to focus head/tail based on shiftKey

Reply:
Good catch!